js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -0,0 +1,208 @@
+
+export const ResourceRoot: React.FC<Props> = (props) => {
+  useTrackPageView();
+
+  const {repoAddress} = props;
+
+  const {resourceName} = useParams<{resourceName: string}>();
+
+  useDocumentTitle(`Resource: ${resourceName}`);
+
+  const resourceSelector = {
+    ...repoAddressToSelector(repoAddress),
+    resourceName,
+  };
+  const queryResult = useQuery<ResourceRootQuery>(RESOURCE_ROOT_QUERY, {
+    variables: {
+      resourceSelector,
+    },
+    fetchPolicy: 'cache-and-network',
+    partialRefetch: true,
+    notifyOnNetworkStatusChange: true,
+  });
+
+  return (
+    <Page>
+      <PageHeader
+        title={
+          <Box flex={{direction: 'row', alignItems: 'center', gap: 12}}>
+            <Heading>{resourceName}</Heading>
+          </Box>
+        }
+      />
+      <Loading queryResult={queryResult} allowStaleData={true}>
+        {({topLevelResourceOrError}) => {
+          if (topLevelResourceOrError.__typename !== 'TopLevelResource') {
+            return null;
+          }
+
+          const configuredValues = Object.fromEntries(
+            topLevelResourceOrError.configuredValues.map((cv) => [cv.key, cv.value]),
+          );
+
+          return (
+            <SplitPanelContainer
+              identifier="explorer"
+              firstInitialPercent={70}
+              firstMinSize={400}
+              height="calc(100% - 60px)"
+              first={
+                <Table>
+                  <thead>
+                    <tr>
+                      <th style={{width: 120}}>Key</th>
+                      <th style={{width: 90}}>Type</th>
+                      <th style={{width: 90}}>Value</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {topLevelResourceOrError.configFields.map((field) => {
+                      const defaultValue = field.defaultValueAsJson;
+                      const actualValue = configuredValues[field.name];
+
+                      const isDefault = defaultValue === actualValue;
+
+                      return (
+                        <tr key={field.name}>
+                          <td>
+                            <Box flex={{direction: 'column', gap: 4, alignItems: 'flex-start'}}>
+                              <strong>{field.name}</strong>
+                              <div style={{fontSize: 12, color: Colors.Gray700}}>
+                                {field.description}
+                              </div>
+                            </Box>
+                          </td>
+                          <td>{remapName(field.configTypeKey)}</td>
+                          <td>
+                            <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
+                              <Tooltip content={<>Default: {defaultValue}</>} canShow={!isDefault}>
+                                {actualValue}
+                              </Tooltip>
+                              {isDefault && <Tag>Default</Tag>}
```

Comment:
Do we need both the tooltip and the tag?

Reply:
The tooltip is useful to view the default value in the non-default case, the default tag is useful to tell without hovering that the value is a default.

We could potentially put the default in a tag (in the non-default case) but the size of the default value could be quite large.